buildspec.yml

```diff
@@ -0,0 +1,8 @@
+version: 0.2
+
+phases:
+
+  build:
+    commands:
+      - chown -R florian:florian .
+      - su florian -c "source /synopsys/scripts/env.sh; echo $PATH"
```

Comment:
Will this command require the runner to provide the password for user `florian`?